tests/src/Simple/HelloWasm/ILHelpers.il

```diff
@@ -46,7 +48,8 @@ ret
     ldloc.0
     box int32
     call void ILHelpers.ILHelpersTest::EatArgs(string, object)
-	ldloc.1
+	  ldloc.1
```

Comment:
Still lots of inconsistent indentation. I think you might have a mix of spaces and tabs, so maybe it looks right in your editor, but it should be spaces.

Reply:
sorry about that, I always need to remember to turn on visual whitespace when editing in VSCode, not really sure where the tabs came from but they're gone now.